std/digest/digest.d

```diff
@@ -579,6 +579,15 @@ interface Digest
         @trusted nothrow void put(scope const(ubyte)[] data...);
 
         /**
+         * This is a conveniance function to pass some strings with char
+         * as element encoding type.
+         */
+        @trusted nothrow final void putText(scope const(char)[] data...)
```

Comment:
OK, compiled it, see that you do indeed need the alias. That should solve the problem.